src/escape_diacritic.rs

```diff
@@ -0,0 +1,147 @@
+#[napi(strict)]
+pub fn escape_diacritic(input: String) -> String {
+  let chars = input.chars();
+  chars.fold("".to_string(), |acc, c| acc + &escape_diacritic_impl(c))
+}
+
+fn escape_diacritic_impl(c: char) -> String {
```

Comment:
`to_string` for every character doesn't sound right… My suggestion is to pass in a `&mut String` and `push_str`/`push` new characters in.